src/_data/sites/en.yml

```diff
@@ -20,6 +20,7 @@ locals:
   docs_latest: latest--docs-eslint.netlify.app
   docs_head: docs-eslint.netlify.app
   docs_next: next--docs-eslint.netlify.app
+  docs_v8: v8-x--docs-eslint.netlify.app
```

Comment:
Because of the dot in the branch name, the subdomain on Netlify is `v8-x--docs-eslint.netlify.app`, not `v8.x--docs-eslint.netlify.app`.